### DISCUSSION TITLE
Max Cadence for bbs02

### DISCUSSION THREAD

**Discussion by Sitebuilder33** [danielnilsson9/bbs-fw#227] [Q&A]
I am curious about the max cadence. Max Cadence for bbshd is around 150rpm I have heard. But not sure how high I dare to set it on the upper gears of bbs02, inside the firmware settings? Perhaps around 120-130 ? Wht do you guys recommend?

**Comment by WvBijsterveld** [comment]
it’s controller limited so you can’t set it too high.
